ext/random/tests/03_randomizer/basic.phpt

```diff
@@ -0,0 +1,70 @@
+--TEST--
+Random: Randomizer: basic
+--FILE--
+<?php
+
+$engines = [];
+$engines[] = new Random\Engine\CombinedLCG(\random_int(\PHP_INT_MIN, \PHP_INT_MAX));
+$engines[] = new Random\Engine\MersenneTwister(\random_int(\PHP_INT_MIN, \PHP_INT_MAX), MT_RAND_MT19937);
+$engines[] = new Random\Engine\MersenneTwister(\random_int(\PHP_INT_MIN, \PHP_INT_MAX), MT_RAND_PHP);
+$engines[] = new Random\Engine\PCG64(\random_int(\PHP_INT_MIN, \PHP_INT_MAX));
+$engines[] = new Random\Engine\Secure(); 
+$engines[] = new class () implements Random\Engine {
+    public function generate(): string
+    {
+        return \random_bytes(16);
+    }
+};
+class UserEngine implements Random\Engine
+{
+    public function generate(): string
+    {
+        return \random_bytes(16);
+    }
+}
+$engines[] = new UserEngine();
+
+foreach ($engines as $engine) {
+    $randomizer = new Random\Randomizer($engine);
+
+    // getInt
+    for ($i = 0; $i < 1000; $i++) {
+        $result = $randomizer->getInt(-50, 50);
+        if ($result > 50 || $result < -50) {
+            die($engine::class . ': getInt: failure');
+        }
+    }
+
+    // getBytes
+    for ($i = 0; $i < 1000; $i++) {
+        $length = \random_int(1, 1024);
+        if (\strlen($randomizer->getBytes($length)) !== $length) {
+            die($engine::class . ': getBytes: failure.');
+        }
+    }
```

Comment:
```suggestion
    for ($i = 1; $i < 1000; $i++) {
        if (\strlen($randomizer->getBytes($i)) !== $i) {
            die($engine::class . ': getBytes: failure.');
        }
    }
```
I feel that this check would be more useful, if it actually checked the various lengths, instead of a random length.